src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/graphs/AdaptiveChainPruner.java

```diff
@@ -56,11 +56,11 @@ public AdaptiveChainPruner(final double initialErrorProbability, final double lo
         final Multimap<V, Path<V,E>> vertexToGoodOutgoingChains = ArrayListMultimap.create();
 
         for (final Path<V,E> chain : chains) {
-            if (chainLogOdds.get(chain).getRight() >= logOddsThreshold) {
```

Comment:
Right I remember this change. When you do the squash + merge of this branch can you make sure its clearly stated in the commit description that this changes the behavior of the adaptive pruner at low/patchy reference coverage sites? This somewhat of a substantial fix to that method and it will be buried by this commit if we aren't careful.